### ISSUE TITLE
toolkito/larasap V1.2.3 requires illuminate/support 5.6.x|5.7.x -> satisfiable by illuminate

### ISSUE THREAD

**Issue by tisuchi** [toolkito/laravel-social-auto-posting#24]
This package doesn't support Laravel 5.8. 

It will. be better to update this package based on Laravel version. 

```
 Problem 1
    - Conclusion: remove laravel/framework v5.8.7
    - Conclusion: don't install laravel/framework v5.8.7
    - toolkito/larasap V1.2.3 requires illuminate/support 5.6.x|5.7.x -> satisfiable by illuminate/support[5.6.x-dev, 5.7.17, 5.7.18, 5.7.19, 5.7.x-dev, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.17, v5.6.19, v5.6.2, v5.6.20, v5.6.21, v5.6.22, v5.6.23, v5.6.24, v5.6.25, v5.6.26, v5.6.27, v5.6.28, v5.6.29, v5.6.3, v5.6.30, v5.6.31, v5.6.32, v5.6.33, v5.6.34, v5.6.35, v5.6.36, v5.6.37, v5.6.38, v5.6.39, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9, v5.7.0, v5.7.1, v5.7.10, v5.7.11, v5.7.15, v5.7.2, v5.7.20, v5.7.21, v5.7.22, v5.7.23, v5.7.26, v5.7.27, v5.7.28, v5.7.3, v5.7.4, v5.7.5, v5.7.6, v5.7.7, v5.7.8, v5.7.9].
```

**Comment by alexisrazok** [comment]
same issue here
there is a solution without downgrade laravel version?

**Comment by Askancy** [comment]
found any solution? I'm using laravel 5.8

**Comment by tisuchi** [comment]
No solution yet in my side. I have postponed this task. To be honest, it seems that, this package is not maintained regularly.

**Comment by Askancy** [comment]
other packages that do something similar? searching i found nothing...

**Comment by alihesari** [comment]
Fixed #32
